app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        cursor = self._connection.execute(f"SELECT * FROM {self.table_name}")
+        rows = cursor.fetchall()

```

Comment:
no need to create a variable - you can use cursor directly in list comprehension